src/AccordionContainer/AccordionContainer.spec.tsx

```diff
@@ -299,7 +445,7 @@ describe('Accordion', () => {
             );
         });
 
-        it('can remove multiple items at the same time', () => {
+        it('can remove multiple items at the same time in an accordion that allows zero items to be expanded', () => {
```

Comment:
Don't think we need to have these tests which assert that item removal still works depending on the value of an unrelated prop.